patches/7271.diff

```diff
@@ -12,32 +12,38 @@ index 5349a20e09..952709d10b 100644
      );
      private static final String nl = "\n";
  
-@@ -107,18 +108,18 @@ public class PrepareBundles {
+@@ -107,18 +108,20 @@ public class PrepareBundles {
                  if ("@types".equals(module.getFileName().toString())) continue;
                  if ("@esbuild".equals(module.getFileName().toString())) continue;
                  if ("@microsoft".equals(module.getFileName().toString())) continue;
 -                if ("@vscode".equals(module.getFileName().toString())) {
 -                    try (DirectoryStream<Path> sds = Files.newDirectoryStream(module)) {
 -                        for (Path sModule : sds) {
--                            checkModule(sModule, sb, tokens2Projects, project2License, bundlesDir, targetDir, externalDir, binariesList);
+-                            checkModule(sModule, sb, tokens2Projects, project2License, bundlesDir, targetDir, exter
+nalDir, binariesList);
 -                        }
 -                    }
 -                    continue;
 +                if ("eastasianwidth".equals(module.getFileName().toString())) continue;
```

Comment:
are skipping liscence verfication steps for new libs added here?

Reply:
Yes

Reply:
we can add required liscences in netbeans patch